.eslintrc.js

```diff
@@ -0,0 +1,195 @@
+module.exports = {
```

Comment:
This came from using `tslint-to-eslint-config` to port our old `tsilnt-config` to the new system.